src/components/layout/layout-v2.tsx

```diff
@@ -156,7 +101,6 @@ function overrideDarkFromQuery() {
 
 export const LayoutV2 = ({
```

Comment:
Is the legacy layout still used in some page? If not, I guess it's a good time to delete the old layout and  rename this file and component to `layout.tsx`

Reply:
the blog is pending. once migrate we can remove the old layout and rename this one finally.